tests/auth_tests/test_forms.py

```diff
@@ -1143,6 +1143,13 @@ def setUpClass(cls):
         # makes tests interfere with each other, see #11505
         Site.objects.clear_cache()
 
+    def assertEmailMessageSent(self, **kwargs):
+        self.assertEqual(len(mail.outbox), 1)
+        msg = mail.outbox[0]
+        for attr, expected in kwargs.items():
+            self.assertEqual(getattr(msg, attr), expected)
```

Comment:
It might be useful to use `subTest` here too, so we can see all failures rather than just one.

Reply:
Great suggestion, thank you! Adding :100: